src/RpcServer/RpcServer.Blockchain.cs

```diff
@@ -197,14 +197,29 @@ protected virtual JObject GetNextBlockValidators(JArray _params)
         {
             using SnapshotView snapshot = Blockchain.Singleton.GetSnapshot();
             var validators = NativeContract.NEO.GetNextBlockValidators(snapshot);
-            return NativeContract.NEO.GetCandidates(snapshot).Select(p =>
+            var candidates = NativeContract.NEO.GetCandidates(snapshot);
+            if (candidates.Length > 0)
             {
-                JObject validator = new JObject();
-                validator["publickey"] = p.PublicKey.ToString();
-                validator["votes"] = p.Votes.ToString();
-                validator["active"] = validators.Contains(p.PublicKey);
-                return validator;
-            }).ToArray();
+                return candidates.Select(p =>
+                {
+                    JObject validator = new JObject();
+                    validator["publickey"] = p.PublicKey.ToString();
+                    validator["votes"] = p.Votes.ToString();
+                    validator["active"] = validators.Contains(p.PublicKey);
+                    return validator;
+                }).ToArray();
+            }
+            else
+            {
+                return validators.Select(p =>
+                {
+                    JObject validator = new JObject();
+                    validator["publickey"] = p.ToString();
+                    validator["votes"] = 0;
```

Comment:
0 or 100000000 whis is better?

Reply:
0 it's fine for me

Reply:
I think this API should be sync with the result that neo-cli client returns. getnextblockvalidators should return nextblockvalidators, and we can have GetCandidates to return the candidates information.